src/50_Samples_%26_Templates/Comment_Section.html

```diff
@@ -0,0 +1,110 @@
+<!---{
+  "preview": "default",
+  "default": "preview",
+  "draft": true,
+  "experiment": true,
+  "component": "amp-form"
+  }--->
+<!--
+#### Introduction
+
+This sample showcases how to build a comment section in AMP HTML.
```

Comment:
A comment section only makes sense together with a login. We should do this also in the sample. Is amp-access the recommended way to integrate a login?